### DISCUSSION TITLE
Problem while using multi_install.sh

### DISCUSSION THREAD

**Discussion by ilhan-athn7** [sebanc/brunch#1105] [Q&A]
Hi, i am prepared a chromeos installation folder for dual booting, like this;
![resim](https://user-images.githubusercontent.com/80551220/121379521-c6816b80-c94c-11eb-8f2a-ff3c5ef19e61.png)

I have problem while executing script, (as highlighted);
![resim](https://user-images.githubusercontent.com/80551220/121379861-18c28c80-c94d-11eb-81ba-bf53bebc5199.png)

I am tried to rename hatch_recovery.bin to rammus_recovery.bin, due to error message show that.
(i have to use hatch, because i have 10.gen CPU, and hatch/volteer support it)
But now i get this error;
![resim](https://user-images.githubusercontent.com/80551220/121380274-6808bd00-c94d-11eb-84d5-9c189cefd403.png)
(meanwhile it is 32 GB partition, i guess it should be more than enough)

So, i am just stuck here, and i need help.
(please let me know if this is wrong place to ask)

**Comment by ilhan-athn7** [comment]
Today i am tried to install on my SDcard with your instructions, 
But i experinced this problem;

> ilhan@NIRVANA-NB-X400:/media/ilhan/Windows/Chrome OS$ sudo bash chromeos-install.sh -src hatch_recovery.bin -dst /dev/mmcblk0
All data on device /dev/mmcblk0 will be lost, are you sure ? (type yes to continue) yes
umount: /dev/mmcblk0: not mounted.
umount: /dev/mmcblk0p10: not mounted.
umount: /dev/mmcblk0p11: not mounted.
umount: /dev/mmcblk0p12: not mounted.
umount: /dev/mmcblk0p2: not mounted.
umount: /dev/mmcblk0p3: not mounted.
umount: /dev/mmcblk0p4: not mounted.
umount: /dev/mmcblk0p5: not mounted.
umount: /dev/mmcblk0p6: not mounted.
umount: /dev/mmcblk0p7: not mounted.
umount: /dev/mmcblk0p8: not mounted.
umount: /dev/mmcblk0p9: not mounted.
WARNING: Primary GPT header is invalid
WARNING: Secondary GPT header is invalid
A5C95C58-8C0F-804B-95E8-8238A46E096E
       start        size    part  contents
           0           1          PMBR (Boot GUID: A5C95C58-8C0F-804B-95E8-8238A46E096E)
           1           1          Pri GPT header
           2          32          Pri GPT table
    19255304    42528712       1  Label: "STATE"
                                  Type: Linux data
                                  UUID: 70700239-319B-E94F-80EE-0F3F55CF01B2
     2117638       65536       2  Label: "KERN-A"
                                  Type: ChromeOS kernel
                                  UUID: 7AF4F5B6-DF10-A44C-963A-862142F46F96
                                  Attr: priority=15 tries=15 successful=0 
    10866696     8388608       3  Label: "ROOT-A"
                                  Type: ChromeOS rootfs
                                  UUID: E702D505-D2A9-B842-957A-BB17860AF156
     2183174       65536       4  Label: "KERN-B"
                                  Type: ChromeOS kernel
                                  UUID: CC2F2647-E999-4944-973B-A3A7205B55FB
                                  Attr: priority=0 tries=15 successful=0 
     2478088     8388608       5  Label: "ROOT-B"
                                  Type: ChromeOS rootfs
                                  UUID: 9212AA93-DF55-9549-9885-0B217E83D9A1
       16448           1       6  Label: "KERN-C"
                                  Type: ChromeOS kernel
                                  UUID: 5417ACD1-9ADF-0240-9574-51F857745A07
                                  Attr: priority=0 tries=15 successful=0 
       16456     2097152       7  Label: "ROOT-C"
                                  Type: ChromeOS rootfs
                                  UUID: 47DE1F69-C47C-E842-B45E-05E035C6B0B8
     2248712       32768       8  Label: "OEM"
                                  Type: Linux data
                                  UUID: 57D5CBB5-04B0-244D-813B-6FF884DBE012
     2113608           1       9  Label: "reserved"
                                  Type: ChromeOS reserved
                                  UUID: AECA0B16-CC12-E248-A632-837526DF9D29
     2113609           1      10  Label: "reserved"
                                  Type: ChromeOS reserved
                                  UUID: 795F273D-8D61-7645-ABA8-92444569F959
          64       16384      11  Label: "RWFW"
                                  Type: ChromeOS firmware
                                  UUID: 92217872-99B9-EB4E-953E-D31B1BAE1A6A
     2412552       65536      12  Label: "EFI-SYSTEM"
                                  Type: EFI System Partition
                                  UUID: A5C95C58-8C0F-804B-95E8-8238A46E096E
    61784031          32          Sec GPT table
    61784063           1          Sec GPT header
Writing partition 1
14,2MiB 0:00:00 [ 289MiB/s] [=========================================================================================================================>] 100%            
Writing partition 2
32,0MiB 0:00:00 [ 825MiB/s] [=========================================================================================================================>] 100%            
Writing partition 3
2,34GiB 0:00:03 [ 660MiB/s] [=========================================================================================================================>] 100%            
Writing partition 4
32,0MiB 0:00:00 [ 720MiB/s] [=========================================================================================================================>] 100%            
Writing partition 5
pv: write failed: Aygıt üzerinde boş yer yok
Writing partition 6
pv: write failed: Aygıt üzerinde boş yer yok
Writing partition 7
pv: write failed: Aygıt üzerinde boş yer yok
Writing partition 8
pv: write failed: Aygıt üzerinde boş yer yok
Writing partition 9
pv: write failed: Aygıt üzerinde boş yer yok
Writing partition 10
pv: write failed: Aygıt üzerinde boş yer yok
Writing partition 11
pv: write failed: Aygıt üzerinde boş yer yok
Writing partition 12
pv: write failed: Aygıt üzerinde boş yer yok
ChromeOS installed.

("Aygıt üzerinde boş yer yok"=> Not enough space)

**Comment by ilhan-athn7** [comment]
Well, i figured out everythink :-D
Thanks you for making chromeos installation possible :-) .
![image](https://user-images.githubusercontent.com/80551220/121483848-4fdd8000-c9d7-11eb-8f91-2ac8ef2f3768.png)

**Comment by TxJK** [comment]
> (meanwhile it is 32 GB partition, i guess it should be more than enough)


For some goddam reason only 22000 MiB partition is supported. 32 GB won't work.
